src/Components/EditModal/EditSubModal.js

```diff
@@ -30,7 +30,7 @@ class EditSubModal extends Component {
       date: this.state.date,
     };
     this.props.updateSubscription(
-      this.props.profile.id,
+      this.props.profile._id,
```

Comment:
Everything looks good, but whenever I change the property of id to ._id I start getting 404 errors. This is with the updated API repo.